web-ui/src/main/resources/catalog/views/default/templates/recordView/type-series.html

```diff
@@ -76,7 +82,7 @@
       data-ng-if="mdView.current.record.related.uuids"
       data-gn-related="mdView.current.record"
       data-user="user"
-      data-layout="card"
+      data-layout="title"
```

Comment:
@MichelGabriel I would keep the card layout here ?

```suggestion
      data-layout="card"
```